vowpalwabbit/loss_functions.cc

```diff
@@ -292,6 +322,58 @@ class quantileloss : public loss_function
   float tau;
 };
 
+class expectileloss : public loss_function
+{
+public:
+  expectileloss(float& q_) : q(q_) {}
+
+  std::string getType() const override { return "expectile"; }
+  float getParameter() const override { return q; }
+
+  // Expectile loss is closely related to the squared loss, but it's an assymetric function with a quantile parameter
```

Comment:
Please move these comments to line 325 above the class definition.
also typo "asymmetric"

Reply:
Moved the comments and fixed the typo.